### PR TITLE
Normalize ArcGIS for SD

### LINKED ISSUE #526: normalize sd/arcgis
[learn our pipeline: normalize](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Runner-pipeline-stages#normalize)

Blocked by #506

Normalize existing [`.ndjson`](http://ndjson.org/) records into [the Vaccinate The States data schema](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Normalized-Location-Schema).

Read all files in the directory passed as the second argument (`sys.argv[2]`), transform each record into the normalized schema (filling in as much detail as possible), then output them to new files in the directory passed as the first argument (`sys.argv[1]`).

Check the wiki to learn more about the purpose of the normalize stage and how to get set up for development!

### Tips

1. Fetch and parse data for this site before you start developing:
    ```sh
    poetry run vaccine-feed-ingest fetch sd/arcgis
    poetry run vaccine-feed-ingest parse sd/arcgis
    ```

1. While working on your code, run it at any point:
    ```sh
    poetry run vaccine-feed-ingest normalize sd/arcgis
    ```

### Example
[Parsed files for `ak/arcgis`](https://github.com/CAVaccineInventory/vaccine-feed-ingest-results/tree/main/ak/arcgis/parsed) are converted to [normalized files](https://github.com/CAVaccineInventory/vaccine-feed-ingest-results/tree/main/ak/arcgis/normalized)

### PR DESCRIPTION
**Pull request by cdjj**
# 

| Key Details |
|-|
| Resolves #526  |
State: SD |
Site: arcgis |

## Data sample

```json
{"id": "sd_arcgis:3bb3f167d57149ef8027a68a609edc2d_0_4dd5c027-89c2-4644-91fa-e48245a109c3", "name": "WALMART PHARMACY #3853", "address": {"street1": "2791 DAKOTA AVE S", "street2": null, "city": "HURON", "state": "SD", "zip": null}, "location": {"latitude": 44.33447799999999, "longitude": -98.214367}, "contact": [{"contact_type": null, "phone": null, "website": "https://www.walmart.com/cp/1228302", "email": null, "other": null}], "languages": null, "opening_dates": null, "opening_hours": null, "availability": null, "inventory": null, "access": null, "parent_organization": null, "links": null, "notes": null, "active": null, "source": {"source": "sd_arcgis", "id": "4dd5c027-89c2-4644-91fa-e48245a109c3", "fetched_from_uri": "https://sdbit.maps.arcgis.com/apps/instant/nearby/index.html?appid=04a23bbd90aa4eeb9c454c303a5c2f06", "fetched_at": "2021-05-10T08:07:46.561619", "published_at": "2021-05-10T08:07:46.561619", "data": {"attributes": {"EditDate": null, "GlobalID": "4dd5c027-89c2-4644-91fa-e48245a109c3", "OBJECTID": 464, "acceptsinsurance": null, "agencyurl": "https://www.walmart.com/cp/1228302", "comments": null, "email": null, "facebook": null, "facilityarea": null, "facilityid": null, "factype": "Pharmacy", "fulladdr": "2791 DAKOTA AVE S", "imageurl": null, "instagram": null, "layer_id": 0, "municipality": "HURON", "name": "WALMART PHARMACY #3853", "operhours": null, "phone": null, "pubview": "Yes", "service_item_id": "3bb3f167d57149ef8027a68a609edc2d", "status": null, "twitter": null, "waittime": null, "walkins": null}, "geometry": {"spatialReference": {"latestWkid": 4326, "wkid": 4326}, "x": -98.214367, "y": 44.33447799999999}}}}
```

## Before Opening a PR
- [X] I tested this using the CLI (e.g., `poetry run vaccine-feed-ingest <state>/<site>`)
- [X] I ran auto-formatting: `poetry run tox -e lint-fix`

### CODE REVIEW

**Comment by bryanculbertson** [comment]
@cdjj approving and merging, but if you get a chance for a 2nd pass, can you try using the `phonenumber` lib?

File: vaccine_feed_ingest/runners/sd/arcgis/normalize.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+from vaccine_feed_ingest.utils.log import getLogger
+
+logger = getLogger(__file__)
+
+
+def _get_contacts(site: dict) -> Optional[List[schema.Contact]]:
+    contacts = []
+    if site["attributes"]["phone"]:
+        sourcePhone = re.sub("[^0-9]", "", site["attributes"]["phone"])
```
[the remark this answers is not in the document]
Comment (cdjj) [reply vaccine_feed_ingest/runners/sd/arcgis/normalize.py:21]: Yeah, I pulled that from another normalizer and was thinking that there should really be a library for that. :)

**Comment by cdjj** [comment]
Yeah, I might have some time tonight to swap over to `phonenumber`; if not, in the next day or two.  I'll take a pass at all the normalizers to see if there's anything else that could move too.

File: vaccine_feed_ingest/runners/sd/arcgis/normalize.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (cdjj) [reply vaccine_feed_ingest/runners/sd/arcgis/normalize.py:21]: Alright, I've got code that works (https://github.com/cdjj/vaccine-feed-ingest/commit/19a011801c8ca1c0e4ea9917803f436c372da319) but haven't had a chance to convert the normalizers to use it yet.  While `phonenumbers` picks up extensions well, `nc/myspot_gov` parses for other text in its numbers, e.g. "option N" or "press 0".  Thoughts on whether it makes sense to try to get those, too, versus dropping them?

**Pull request merged**